templates/puppet_signscript.rb.erb

```diff
@@ -0,0 +1,43 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+#
+
+require 'net/http'
+require 'net/https'
+require 'fileutils'
+require 'json'
+require 'yaml'
+
+csr = STDIN.read
+
+settings_file    = File.exist?('/etc/puppetlabs/puppet/foreman.yaml') ? '/etc/puppetlabs/puppet/foreman.yaml' : '/etc/puppet/foreman.yaml'
```

Comment:
I think this can just be `/etc/puppetlabs/puppet/foreman.yaml` since we don't support Puppet 3 anymore.